package.json

```diff
@@ -33,15 +33,17 @@
   },
   "devDependencies": {
     "@crave/farmblocks-dev-scaffold": "^1.0.0",
-    "@storybook/addon-actions": "^3.4.6",
-    "@storybook/addon-info": "^3.4.6",
-    "@storybook/addon-links": "^3.4.6",
-    "@storybook/addon-storyshots": "^3.4.6",
-    "@storybook/addons": "^3.4.6",
-    "@storybook/react": "^3.4.6",
+    "@storybook/addon-actions": "^4.1.3",
+    "@storybook/addon-info": "^4.1.3",
+    "@storybook/addon-links": "^4.1.3",
+    "@storybook/addon-storyshots": "^4.1.3",
+    "@storybook/addons": "^4.1.3",
+    "@storybook/react": "^4.1.3",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-eslint": "^10.0.0",
+    "babel-loader": "7",
```

Comment:
Since storybook v4 relies on babel 7, I think this is an opportunity for us to upload to babel 7 as well. Let's try?